src/Relay.php

```diff
@@ -60,6 +61,30 @@ public static function connectionDefinitions(array $config)
     }
 
     /**
+     * Returns a GraphQLObjectType for a connection with the given name,
+     * and whose nodes are of the specified type.
+     *
+     * @param array $config
+     * @return ObjectType
+     */
+    public static function connection(array $config)
```

Comment:
Maybe we should rename this to `connectionType` to make the difference obvious to `connectionDefinitions`